docs/governance/governance-interaction.md

```diff
@@ -192,51 +226,51 @@ https://<gateway>.multiversx.com/vm-values/query
 ```json
 {
   "scAddress": "erd1qqqqqqqqqqqqqqqpqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqrlllsrujgla",
-  "funcName": "viewUserVoteHistory",
-  "args": ["<address>"]
+  "funcName": "viewDelegatedVoteInfo",
+  "args": ["<proposal_nonce-hex>", "<address>"]
 }
 ```
 
-the `address` represents the address in hex format. The response will contain the following json definition where all fields are base64-encoded:
+- `proposal_nonce` → the proposal identifier (nonce, hex-encoded).  
+- `address` → the bech32 address of the account to check.  
+
+> **Note:** The older function `viewUserVoteHistory` (which returned lists of proposal nonces) is now considered legacy.  
+> Use `viewDelegatedVoteInfo` for detailed voting power and stake information.
+
+The response will contain the following json definition where all fields are base64-encoded:
 
 ```json
 {
   "returnData": [
-    "<the number of delegated nonces>",
-    "<delegated nonce 0>",
-    "<delegated nonce 1>",
-    ...
-    "<delegated nonce n>",
-    "<the number of direct nonces>",
-    "<direct nonce 0>",
-    "<direct nonce 1>",
-    ...
-    "<direct nonce m>"
+    "<used_power>", (voting power already used by this address on the proposal)
+    "<used_stake>", (stake associated with the used power)
+    "<total_power>", (total available voting power for the address)
+    "<total_stake>" (total stake considered in governance for the address)
   ]
 }
 ```
 
-Example for an address that cast votes on 5 proposals:
+Example for an address that voted:
 ```json
 {
   "returnData": [
-    "Aw==", (number of delegated nonces: 3)
-    "AQ==", (nonce: 1)
-    "Ag==", (nonce: 2)
-    "Aw==", (nonce: 3)
-    "Ag==", (number of direct nonces: 2)
-    "BA==", (nonce: 4)
-    "BQ==", (nonce: 5)
+    "Cg==", (used power: 10)
+    "ZAA=", (used stake: 100)
+    "A+g=", (total power: 1000)
+    "Gg4M=", (total stake: 10000)
   ]
 }
 ```
+In this example, the queried address voted on this proposal with **100 stake**, which translated into **10 voting power**. The proposal overall had **10000 total stake** and **1000 total voting power** recorded.
```

Comment:
right now we have a linear voting system. We moved from a quadratic voting system where each stake translated to sqrt(stake). EG: 100 stake -> 10 voting power. 
please change the values to be equal: 
In this example, the queried address voted on this proposal with **100 stake**, which translated into **100 voting power**. The proposal overall had **1000 total stake** and **1000 total voting power** recorded.